source/release-notes/1.6.txt

```diff
@@ -0,0 +1,145 @@
+.. _c2c-release-notes-1.6:
+
+===================================
+Release Notes for mongosync 1.6
+===================================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.6.0-c2c-release-notes:
+
+1.6.0 Release
+-------------
+
+**September 22, 2023**
+
+Exclusion Filters
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.6.0, the :ref:`c2c-api-start` API endpoint now supports the
+use of exclusion filters through the ``excludeNamespaces`` parameter.
+
+For more information, see :ref:`c2c-filtered-sync`.
+
+Regular Expression Filters
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Both inclusion and exclusion filters in :ref:`c2c-filtered-sync` now 
+support matching databases and collections using Regular Expressions 
+with the ``databaseRegex`` and ``collectionRegex`` fields.
+
+For more information, see :ref:`c2c-filter-regex`.
+
+Exclusion Filters
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.6.0, the :ref:`c2c-api-start` API endpoint now supports the
+use of exclusion filters through the ``excludeNamespaces`` parameter.
+
+For more information, see :ref:`c2c-filtered-sync`.
+
+Regular Expression Filters
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Both inclusion and exclusion filters in :ref:`c2c-filtered-sync` now 
+support matching databases and collections using Regular Expressions 
+with the ``databaseRegex`` and ``collectionRegex`` fields.
+
+For more information, see :ref:`c2c-filter-regex`.
+
+7.0 Support
+~~~~~~~~~~~
+
+Starting in 1.6.0, ``mongosync`` supports sync with MongoDB 7.0.
+
+Other Notes
+~~~~~~~~~~~
+
+New Features:
+
+- Sped up the commit process by parallelizing metadata cleanup. 
+
+- Reduced memory allocation during change event application by approximately 
+  30% for pre-6.0 releases of MongoDB Server versions.
+
+- Changed telemetry to also send error messages to Segment when ``mongosync``
+  exits with a fatal error.
+
+- Added support for ``mongosync`` to sync between source and destination 
+  clusters that have the same major and minor versions, but different 
+  patch versions.
+
+- Upgraded build to use Go 1.20.
+
+- ``mongosync`` now periodically logs when server operations take a longer 
+  than expected time.
+
+Issues Fixed:
+
+- Fixed a bug introduced in 1.5, where ``mongosync`` could fail to apply change
+  events for a document if they occur shortly after the document is copied
+  during the Collection Copy phase.
```

Comment:
The team would like to change this bullet to the following:

> Fixed a bug introduced in Mongosync v1.5.0 and discovered by our internal testing where in an unlikely edge case Mongosync can miss a change event to be applied during the Collection Copy phase. The change event must fall between a specific `_id` range that Mongosync is in the process of copying, must occur and be processed while that specific `_id` range is being copied, and must not be followed by another change event for the same `_id` throughout the rest of the migration. This bug can only occur if Mongosync is processing change events with near zero replication lag during Collection Copy.

Reply:
Fixed